src/grpc_subscription_autoreconnect_streams.rs

```diff
@@ -45,21 +46,18 @@ pub fn create_geyser_reconnecting_stream(
                         log!(if attempt > 1 { Level::Warn } else { Level::Debug }, "Connecting attempt #{} to {}", attempt, addr);
                         async move {
 
-                            let connect_result = GeyserGrpcClient::connect_with_timeout(
-                                    addr, token, config,
-                                    connect_timeout,
-                                    request_timeout,
-                                false)
-                                .await;
-                            let mut client = connect_result?;
+                            let mut builder =  GeyserGrpcClient::build_from_shared(addr).unwrap()
+                                .x_token(token).unwrap()
+                                .connect_timeout(connect_timeout.unwrap_or(Duration::from_secs(10)))
+                                .timeout(request_timeout.unwrap_or(Duration::from_secs(10)))
+                                .tls_config(config.unwrap_or(ClientTlsConfig::new())).unwrap();
```

Comment:
There is no tls config, check it again if new tls config means empty.

Reply:
I ran the examples and they work without tls